OpenIPSL/Electrical/Machines/PSSE/CIM5.mo

```diff
@@ -0,0 +1,173 @@
+within OpenIPSL.Electrical.Machines.PSSE;
+model CIM5 "PSSE CIM5 three-phase induction motor model"
+  extends OpenIPSL.Electrical.Machines.PSSE.BaseClasses.baseMotor;
+  import Modelica.Constants.eps;
```

Comment:
```suggestion
```
No need for this import use `C.eps` since there is a top-level import, check it out.